vaadinfrontend/src/main/resources/application.properties

```
@@ -30,6 +30,7 @@ spring.mail.port=${MAIL_PORT:587}
 service.host.name=${DM_SERVICE_HOST:localhost}
 service.host.protocol=${DM_HOST_PROTOCOL:https}
 service.host.port=${DM_SERVICE_PORT:-1}
+service.host.proxy-path=${DM_SERVICE_PROXY_PATH:}
```

I'd suggest just moving the servlet context path definition into [application properties](https://www.baeldung.com/spring-boot-context-path) instead
```suggestion
server.servlet.context-path=${DM_SERVICE_SERVLET_CONTEXT:}
```